terraform/modules/paas/toolbox.tf

```diff
@@ -9,8 +9,8 @@ resource "cloudfoundry_app" "toolbox" {
 }
 
 resource "cloudfoundry_route" "toolbox" {
-  domain   = data.cloudfoundry_domain.external.id
-  hostname = "toolbox${var.external_hostname_suffix}"
+  domain   = data.cloudfoundry_domain.paas_external.id
+  hostname = "toolbox${var.internal_hostname_suffix}"
```

Comment:
It looks kind of confusing having `paas_external` and the `internal_hostname_suffix` - I guess this is because the internal suffix is more "global", because it's a shared domain?

Probably fine for this PR, but I wonder if we want a better name for this variable (e.g. like `global_hostname_suffix`)?

Reply:
I'd agree, although i'd rather not introduce another variable for this single url. Since we just need to namespace by the space, we could simply use `${var.space}?`

Reply:
Nah, I think let's leave it with `internal_hostname_suffix`. Just wanted to check.